setup.py

```diff
@@ -30,7 +30,7 @@
 except ImportError:
     pass
 
-requires = ['numpy >=1.13', 'pillow', 'pyresample >=1.24.0', 'trollsift',
+requires = ['numpy >=1.13', 'pillow>=9.2.0', 'pyresample >=1.24.0', 'trollsift',
```

Comment:
Why is pillow 9.2.0+ needed? That version isn't even 6 months old so if it isn't required it'd be nice to not force it here.